jinahub/encoders/image/TimmImageEncoder/timm_encoder.py

```diff
@@ -0,0 +1,100 @@
+from typing import Dict, Iterable, Optional, Tuple
+
+import numpy as np
+import torch
+import torchvision.transforms as T
+from jina import DocumentArray, Executor, requests
+from jina_commons.batching import get_docs_batch_generator
+from timm import create_model
+from timm.data import resolve_data_config
+from timm.data.transforms_factory import create_transform
+
+
+class TimmImageEncoder(Executor):
+    """
+    :class:`TimmImageEncoder` encodes ``Document`` blobs of type `ndarray`
+    (`float32`) and shape `H x W x 3` into `ndarray` of `D`.
+    Where `D` is the Dimension of the embedding.
+    Input image in Document should be in RGB format.
+
+    If `use_default_preprocessing=False`, the expected input shape is
+    `3 x H x W` with `float32` dtype.
+
+    Internally, :class:`TimmImageEncoder` wraps the pre-trained models from
+    `Timm library`.
+    https://rwightman.github.io/pytorch-image-models/
+
+    :param model_name: the name of the model. Models listed on:
+        https://rwightman.github.io/pytorch-image-models/models/
+    :param device: Which device the model runs on. For example 'cpu' or 'cuda'.
+    :param default_traversal_paths: Defines traversal path through the docs.
+        Default input is the tuple ('r',) and can be overwritten.
+    :param default_batch_size: Defines the batch size for inference on the loaded Timm model.
+        Default batch size is 32 and can be updated by passing an int value.
+    :param use_default_preprocessing: Should the input be preprocessed with default configuration.
+    :param args:  Additional positional arguments.
+    :param kwargs: Additional keyword arguments.
+    """
+
+    def __init__(
+        self,
+        model_name: str = "resnet18",
+        device: str = "cpu",
```

Comment:
```suggestion
        device: str = 'cpu',
```

We use single quotes - can you also check other lines in this file.

Reply:
If you are using black manually, remember to use `black -S`, so that it won't default to double quotes

Reply:
Yes, that was the problem. black was replacing the single quotes with double quotes.